contributing_to_packages/package_guidelines_required.rst

```diff
@@ -208,46 +208,46 @@ So, don't do this:
 
     def build(self):
         cmake = CMake(self)
-		cmake.configure()
-		cmake.build()
-		cmake.install()
+        cmake.configure()
+	cmake.build()
```

Comment:
are you using tabs instead of spaces? some lines are not aligned.

Reply:
I've fixed it, tabs and spaces were mixed before, i've fixed it too